apps/ima/sections/summary-data-sections.js

```diff
@@ -191,7 +191,7 @@ module.exports = {
         step: '/harm-claim-summary',
         field: 'sih-countries',
         parse: (list, req) => {
-          if (!req.sessionModel.get('steps').includes('/harm-claim-summary')) {
+          if (req.sessionModel.get('sih-countries') !== undefined && req.sessionModel.get('sih-countries').length > 0) {
```

Comment:
Isn't this saying if there are sih-countries return null? Shouldn't it be the opposite i.e. if there are no sih-countries/sih-countries is undefined  then return null?

Reply:
Changed as suggested and works fine when re-tested.